odooghost/services/base.py

```diff
@@ -20,10 +20,14 @@
 
 
 class BaseService(abc.ABC):
-    def __init__(self, name: str, stack_config: "StackConfig") -> None:
-        self.name = name
+    def __init__(self, stack_config: "StackConfig") -> None:
         self.stack_config = stack_config
         self.stack_name = stack_config.name
+        self._check_name_attribute()
```

Comment:
why this ? why removing name from service subclasss constructor ?

Reply:
I did that to be able to use the function ac_stacks_services with BaseService.list...